python/tests/test_strategies.py

```diff
@@ -20,14 +21,16 @@ def test_backtest():
     """
     Ensures that the backtest function works on all the registered strategies, with their default parameter values
     """
-    current_date = datetime.today().strftime("%Y-%m-%d")
     sample = pd.read_csv(SAMPLE_CSV, parse_dates=["dt"])
+    # Simulate custom indicator
+    sample["custom"] = np.random.random((sample.shape[0],)) * 100
+
     for strategy in STRATEGY_MAPPING.keys():
         if strategy == "sentiment":
-            data = get_yahoo_data("TSLA", "2020-01-01", current_date)
+            data = get_yahoo_data("TSLA", "2020-01-01", "2020-07-04")
             sentiments = get_bt_news_sentiment(keyword="tesla", page_nums=2)
             cerebro = backtest(
-                strategy, data, sentiments=sentiments, senti=0.4
+                strategy, data, sentiments=sentiments, senti=0.4, plot=False
```

Comment:
any reasons why `plot` was set to `False`?

Reply:
Ah set it to false so it's consistent with the rest of the strategies. Also, this makes sure it doesn't pause the test when I'm testing locally. Any reason why it was set to True? 😄

Reply:
Ah iirc it was the default XD

Reply:
Ah yes, it's the default, but i think makes sense to set just this one to non-default for reason above. I see your point though, perhaps we can argue to set all the plot=True in a separate PR later if we decide makes sense :)